src/popup/popup.html

```diff
@@ -90,13 +90,13 @@ <h1>Add new profile</h1>
                 </fieldset>
 
                 <fieldset>
-                    <label for="idp">IDP</label>
+                    <label for="idp">Identity provider</label>
                     <input id="idp" name="idp" type="text" placeholder="Solid identity provider"/>
```

Comment:
This is not updated.

Reply:
Do you mean the placeholder? Do you want it to also be "Identity provider" instead of "Solid identity provider"?

Reply:
I said in my initial comment that I wanted it to be consistent with the placeholder for WebID.

Reply:
✔️